src/backend/distributed/operations/worker_split_copy_udf.c

```diff
@@ -244,6 +253,10 @@ CreatePartitionedSplitCopyDestReceiver(EState *estate,
 		shardIntervalToSplitCopy->relationId);
 	char partitionMethod = cacheEntry->partitionMethod;
 	Var *partitionColumn = cacheEntry->partitionColumn;
+	if (partitionColumn == NULL)
```

Comment:
in https://github.com/citusdata/citus/pull/6087/files#diff-7850227231c0f02beb16729d6da987cc50c6a56946bb0076ec787fc9bc1c86d6R250 this code has been removed because we need to be able to split a table without a partition column